### ISSUE TITLE
when using pylance code completion, press "enter” will replace the code wrote behind it.

### ISSUE THREAD

**Issue by Kndy666** [microsoft/PTVS#7132]
**Describe your question**
The action pylance code completion made is just like enabling the INSERT and enter new codes.It will replace the words behind the position you use the code completion.The first screenshot displays that I have selected the option before I press the enter.The second screenshot displays that after I press the enter, the bracket I have wrote before disappeared.

**What have you tried?**
I have no ideas whether it’s a bug or just I haven’t set current settings of pylance.At present, I just disabled the code completion.
![2022-08-21 152522](https://user-images.githubusercontent.com/35413282/185783742-024971d1-c57c-427a-beca-ed5abf3fecfc.png)
![2022-08-21 152540](https://user-images.githubusercontent.com/35413282/185783744-7339b6bd-6b9c-4274-a715-c7b9b8c4f7da.png)

**Comment by TheRealQuantam** [comment]
This is a problem I run into constantly. I was considering whether to make an issue for it. Honestly autocomplete in Visual Studio is pretty much useless in Python due to this and one other issue. My standard behavior when the autocomplete window pops up is to just hit ESC, as I know it won't work properly.

**Comment by Kndy666** [comment]
Yes.The first time I met this problem,as usual,I pressed ESC to close the menu.The problem didn't exist when I use Visual Studio at version 2019 with language server Intellisense.But this version has more acute problems such as high cpu usage.On second thought,maybe python in visual studio isn't that welcome.

**Comment by TheRealQuantam** [comment]
IME autocomplete has never worked in VS/Python. In older versions it had the issue where you partially type in a name, then select something from the autocomplete box and it won't fully replace the partial name, leaving you with the autocomplete text plus a letter or two of what you typed, creating an invalid name that you don't notice until you run it and it fails.

**Comment by StellaHuang95** [comment]
Similar issue was also mentioned in #6789.
